more/free-programming-cheatsheets.md

```diff
@@ -255,6 +255,7 @@
 * [Python Regex Cheatsheet](https://www.debuggex.com/cheatsheet/regex/python) - Debuggex (HTML)
 * [Python WorldWide Cheatsheet](https://cheatography.com/davechild/cheat-sheets/python/) - Cheatography (Python)
 * [Scapy Cheat Sheet](https://www.templateroller.com/template/160817/scapy-cheat-sheet-jeremy-stretch.html) - Jeremy Stretch (PDF)
+* [Python Begineers Cheatsheet](https://hakin9.org/wp-content/uploads/2020/02/beginnersPythonCheatSheet.pdf) - No Starch (PDF)
```

Comment:
is there no link from the author/publisher?

Reply:
Nope, i tried to find on where it was originally uploaded but got nothing except hacki9.org.

Reply:
Eric Mattes is the right author.

![image](https://user-images.githubusercontent.com/3125580/193810225-7c8d5b34-a7e9-4d95-9e40-8d4364dd577a.png)